x/ccv/provider/keeper/genesis.go

```diff
@@ -103,6 +105,50 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 
 	k.SetParams(ctx, genState.Params)
 	k.InitializeSlashMeter(ctx)
+
+	// Set the last provider consensus validator set
+	k.SetLastProviderConsensusValSet(ctx, genState.LastProviderConsensusValidators)
```

Comment:
This action seems redundant given that InitGenesisValUpdates will set the valset to something else.